client/src/identityServerDebug.ts

```diff
@@ -264,9 +264,11 @@ export class IdentityServerDebugSession extends LoggingDebugSession {
             const answer = this.iamRemoteRuntime.fetchVariables(response, args, request);
             answer.then((remoteResponse) => {
                 remoteResponse.body.variables.forEach((element) => {
-
-                    if (element.name === DebugConstants.SAML_REQUEST
-                        && !element.value.includes("NO SAML Request Added")) {
+                    //TODO comparing the value
+                    // Check whether it is unefined.
+                    if (DebugConstants.SAML_REQUEST === element.name
```

Comment:
Please resolve this todo since it's a small change.

Reply:
Fixed